app/lib/frontend/templates.dart

```diff
@@ -888,3 +890,11 @@ const _schemaOrgSearchAction = const {
     'query-input': 'required',
   },
 };
+
+Map _schemaOrgPkgMeta(PackageVersion pv) => {
```

Comment:
Happy to start small, but would you add TODOs for

http://schema.org/programmingLanguage - could start w/ text, but have fun with http://schema.org/ComputerLanguage
http://schema.org/codeRepository - if the homepage is github.com
http://schema.org/dateModified - last published
http://schema.org/dateCreated - first published
http://schema.org/license
http://schema.org/version

Reply:
Updated these except for the codeRepository (added a TODO for it). I have a few plans around the github / gitlab link handling, it will be done eventually...